releasenotes/notes/go-interface-for-integration-logs-671454384a93c1ce.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    This change adds a Go interface for logs coming from integrations. The RTLoader will be able to use this interface to send logs directly to the Agent from integrations.
```

Comment:
Not sure we need a changelog for this PR. This does not provide any user impacting behavior change. I'd save the changelog for when the feature is "done" and usable (even if behind a config flag).